### PR TITLE
Updated syntax of html

### LINKED ISSUE #8340: content: Typo in example.
### What page needs to be updated?

[Control focus with tabindex](https://web.dev/control-focus-with-tabindex/)

### Why is this update needed?
There is a typo in the example of section: **_Create accessible components with "roving tabindex"_**.

`<div role="toolbar">`
  `<button tabindex="-1">Undo</div>`
  `<button tabindex="0">Redo</div>`
  `<button tabindex="-1">Cut</div>`
`</div>`

the child elements of the `div` should be closed with a `button` instead of `div`.

### What's the deadline?
None

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
Fixes #8340 

Changes proposed in this pull request:

- Updated the syntax of the button tag

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/content/en/accessible/control-focus-with-tabindex/index.md [hunk 1]
```diff
@@ -133,9 +133,9 @@ method on it.
 
 ```html/2-3
 <div role="toolbar">
-  <button tabindex="-1">Undo</div>
-  <button tabindex="-1">Redo</div>
-  <button tabindex="0">Cut</div>
+  <button tabindex="-1">Undo</button>
```
Comment (dush-ik) [inline src/site/content/en/accessible/control-focus-with-tabindex/index.md:136]: There is one more at line number 126,127,128.

Comment (Abhiram-Gullapalli) [reply src/site/content/en/accessible/control-focus-with-tabindex/index.md:136]: Oh, I must have overlooked it.

**Comment by Abhiram-Gullapalli** [comment]
Could you check it again @dush-ik

**Comment by Abhiram-Gullapalli** [comment]
I'm sorry for what, am I doing something wrong?

**Comment by dush-ik** [comment]
Looks good. I`m not able to approve it though.

**Comment by Abhiram-Gullapalli** [comment]
I was experiencing a similar glitch, I couldn't re-request review

**Review by rachelandrew — APPROVED** [review]
thank you!

**Comment by Mzul86** [comment]
Tq support me

**Comment by Mzul86** [comment]
I hope u support me web

**Pull request merged**